package.json

```diff
@@ -44,10 +42,12 @@
         "setup": "pnpm install",
         "start": "pnpm build --stream --filter=coveo-styleguide && pnpm -r start --parallel --filter=\\!react-vapor",
         "test": "pnpm -r test --parallel",
+        "cy:open": "pnpm -r cy:open --stream",
+        "cy:run": "pnpm -r cy:run --stream",
```

Comment:
Should these really be on the root package.json? Maybe we could have only one of them like `test:e2e`

Reply:
It would be easier for dev/jenkins to start the command if it's on the root package

What do you think of `e2e:open` and `e2e:run` ?

Reply:
What's the use-case for opening but not running?

Reply:
it opens a window where you can choose what you run, you see the browser run, etc. 

Jenkins will run `e2e:run`
a dev working on e2e tests will probably run `e2e:open`
a dev working on react-vapor might run `e2e:run` to make sure they didn't break anything